modulo5/LabECommerce/.gitignore

```
@@ -0,0 +1,144 @@
+.env
+.DS_STORE
+node_modules
+scripts/flow/*/.flowconfig
+.flowconfig
+*~
+*.pyc
+.grunt
+_SpecRunner.html
+__benchmarks__
+build/
+remote-repo/
+coverage/
+.module-cache
+fixtures/dom/public/react-dom.js
+fixtures/dom/public/react.js
+test/the-files-to-test.generated.js
+*.log*
+chrome-user-data
+*.sublime-project
+*.sublime-workspace
+.idea
+*.iml
+.vscode
+*.swp
+*.swo
+
+packages/react-devtools-core/dist
+packages/react-devtools-extensions/chrome/build
+packages/react-devtools-extensions/chrome/*.crx
+packages/react-devtools-extensions/chrome/*.pem
+packages/react-devtools-extensions/firefox/build
+packages/react-devtools-extensions/firefox/*.xpi
+packages/react-devtools-extensions/firefox/*.pem
+packages/react-devtools-extensions/shared/build
+packages/react-devtools-extensions/.tempUserDataDir
+packages/react-devtools-inline/dist
+packages/react-devtools-shell/dist
+packages/react-devtools-timeline/dist
```

Se voce quer ignorar todos os packages que começam com `react-devtools` pode simplificar para 
```suggestion
packages/react-devtools-*
```
Mas o ideal seria manter nesse `.gitignore` somente coisas relacionadas a essa API e que realmente existem dentro da pasta`modulo5/LabECommerce` na sua maquina local, e retirar coisas extras de outros projetos desse repositorio como essas pastas do react, vue, etc.